src/gg.regression.unity.bots/Runtime/Scripts/GenericBots/RGMonkeyBot.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System.Collections.Generic;
+using RegressionGames.ActionManager;
+using UnityEngine;
+
+namespace RegressionGames.GenericBots
+{
+    public class RGMonkeyBot : MonoBehaviour, IRGBot
+    {
+        public float actionInterval = 0.0f; // every frame by default
```

Comment:
I could see arguments either way, but I'd make this something like 0.1f by default.  That way the tester can at least see what is happening.

Reply:
I was going back and forth on this, since the faster interval generally covers more functionality faster. But the longer interval has some advantages for some of the games too. I've set it to 0.1f.